posnums.c

```diff
@@ -0,0 +1,95 @@
+/*Ashank Kumar find average, if numbers are prime and if they are relatively prime*/
```

Comment:
add more comments throughout so that when things like this happen, I can easily find where you went wrong.